docs/source/models/concept_models/vivarium_mncnh_portfolio/mncnh_interventions.rst

```diff
@@ -0,0 +1,130 @@
+
+
+Calibration Strategy
+--------------------
+
+The following decision tree shows all of the paths from delivery facility choice to CPAP availability.  Each path corresponds to a disjoint event that we can sum over to find the population probability of RDS mortality.
+
+.. graphviz::
+
+    digraph CPAP {
+        rankdir = LR;
+        start [color="none"]
+        facility [label="1"]
+        home [label="p_RDS_without_CPAP"]
+        BEmONC [label="7"]
+        CEmONC [label="7"]
+        BEmONC_wo [label="p_RDS_without_CPAP"] 
+        BEmONC_w [label="p_RDS_with_CPAP"]
+        CEmONC_wo [label="p_RDS_without_CPAP"] 
+        CEmONC_w [label="p_RDS_with_CPAP"]
+
+        start -> facility  [label = "delivery facility type"]
+        facility -> home  [label = "home birth"]
+        facility -> BEmONC  [label = "BEmONC"]
+        facility -> CEmONC  [label = "CEmONC"]
+
+        BEmONC -> BEmONC_w  [label = "CPAP available"]
+        BEmONC -> BEmONC_wo  [label = "CPAP unavailable"]
+
+        CEmONC -> CEmONC_w  [label = "CPAP available"]
+        CEmONC -> CEmONC_wo  [label = "CPAP unavailable"]
+    }
+
+.. math::
+    \begin{align*}
+        p_\text{RDS} 
+        &= \sum_{\text{paths without CPAP}} p(\text{path})\cdot p_\text{RDS_wo}\\
+        &+ \sum_{\text{paths with CPAP}} p(\text{path})\cdot p_\text{RDS_w}\\[.1in]
+        p_\text{RDS_w} &= RR_\text{CPAP} \cdot p_\text{RDS_wo}
+    \end{align*}
+
+
+
+When we fill in the location-specific values for delivery facility rates, CPAP coverage, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_RDS_w and p_RDS_wo), which we can solve analytically.
+
+Scenarios
+---------
+
+The :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>` includes specific scenarios that might change the importance of this intervention. This includes interventions that might lower the prevalence of RDS (like ACS), and might also include interventions that increase the coverage of CPAP or increase the rate of preterm delivery with access CPAP.
+
+
+Assumptions and Limitations
+---------------------------
+
+- We assume that CPAP availability captures actual use, and not simply the machine being in the facility 
+- We assume that the delivery facility is the final facility in the care continum for deliveries that are transferred due to complications
+- We assume that the relative risk of RDS mortality with CPAP in practice is similar to that found in the Cochrane Review meta-analysis
+
+Validation and Verification Criteria
+------------------------------------
+
+- Population-level mortality rate should be the same as when this intervention is not included in the model
+- The ratio of RDS mortality rate among those without CPAP access divided by those with CPAP access should equal the relative risk from the Cochrane Review
```

Comment:
This seems unlikely to be true due to confounding with facility choice?

Reply:
@zmbc What do you mean? Do you mean that P(die of RDS | have RDS) is likely to be lower in facilities than in home births, even if the facility doesn't have a CPAP machine?

@aflaxman Can you be more specific when you talk about "mortality rate"? Do you mean rates per person-time, or probabilities (i.e., mortality risk)? Does the Cochrane review tell us the relative risk for the case fatality rate (i.e., the conditional probability I wrote above), or the overall mortality rate (or risk) of RDS among preterm babies? Or among all newborns? What population is in the denominator?

Reply:
I think it cancels out, since it is a ratio, but I will write it out precisely